dev-python/pygls/pygls-0.10.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="A pythonic generic language server"
+HOMEPAGE="https://github.com/openlawlibrary/pygls"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
```

Comment:
`distutils_enable_tests` already automatically adds the IUSE and RESTRICT lines for you

Reply:
Thanks, fixed.